src/App/Containers/Authentication/Login/LoginPage.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
import styles as global to avoid conflicts from other css or any bootstrap version